managers/views.py

```diff
@@ -74,6 +75,24 @@ def get_queryset(self) -> QuerySet:
 
         return projects_queryset
 
+    def post(self, _request: HttpRequest) -> HttpResponseRedirect:
+        project_id_to_change_status = self.request.POST.get("project-status-change")
+
+        if project_id_to_change_status is not None:
+            self._manage_project_status(project_id_to_change_status)
+
+        return redirect("custom-projects-list")
+
+    @staticmethod
+    def _manage_project_status(project_id: int) -> Project:
+        project = Project.objects.get(id=project_id)
+        project_status = project.suspended
+
+        project.suspended = not project_status
+        project.save()
```

Comment:
Please add `updated_fields` parameter to save method and add only those fields which you're updating here

Reply:
Added